Fix #11354: Coding style causes undefined behaviour

sohamroy19 opened the pull request:
Referring #11354, an underscore followed by a capital letter used as a prefix might cause undefined behaviour.
Replaced variable or typename prefixes of type _T([A-Z]) with _T$1, essentially.

duncanspumpkin commented on src/openrct2/drawing/Font.h:
```diff
@@ -33,8 +33,8 @@ enum class FontSpriteBase : int16_t
 
-struct _TTF_Font;
-using TTF_Font = _TTF_Font;
+struct Internal_TTF_Font;
```
This is the wrong style. Should be `InternalTTFFont`

sohamroy19 replied:
Should be fixed by https://github.com/OpenRCT2/OpenRCT2/pull/15521/commits/fda4476288d28fca3b4b0b00e6a874ca4920bd29 now.

ZehMatt commented:
If you like you can also add yourself to the ```contributors.md``` file given this is your first PR.

sohamroy19 commented:
Added myself to "Bug fixes" section in `contributors.md`, also fixed the build failure of https://github.com/OpenRCT2/OpenRCT2/pull/15521/commits/fda4476288d28fca3b4b0b00e6a874ca4920bd29.

tupaschoal reviewed (approved):
LGTM

Gymnasiast commented:
The error was in the commit message. You need to leave one blank line between the title and the body. Since it’s your first time and I could easily fix it when squash merging, I have fixed it for you.

sohamroy19 commented:
Doesn't the CI run message [here](https://github.com/OpenRCT2/OpenRCT2/runs/3796128524#step:4:18) show that there indeed was a blank line between the title and the body? I did notice this error but I couldn't (still can't) understand what it wants.
I didn't mention this as it was non-essential, the now-fixed build error I was referring to earlier is [this](https://github.com/OpenRCT2/OpenRCT2/runs/3792834251).
